ios/FluentUI/List/ListItem.swift

```diff
@@ -106,6 +106,8 @@ public struct ListItem<LeadingContent: View,
         var backgroundView: some View {
             if let backgroundColor = backgroundStyleType.defaultColor(tokenSet: tokenSet) {
```

Comment:
`defaultColor` sounds like a function that should always return a non-optional color. Is there a scenario in which we would want it to return nil?

Reply:
It looks like the decision was made that if a client wanted to override the background color **not** using the token system, the color would be nil.

I would assume that having the default for `.custom` background style to be `.clear` could cause issues with someone attempting to override it outside the token system?


``` swift
// Different background color is used for `TableViewCell` by getting the appropriate tokens and integrate with the cell's `UIBackgroundConfiguration`
@objc(MSFTableViewCellBackgroundStyleType)
public enum TableViewCellBackgroundStyleType: Int {
    // use for flat list of cells
    case plain
    // use for grouped list of cells
    case grouped
    // clear background so that TableView's background can be shown
    case clear
    // in case clients want to override the background on their own without using token system
    case custom

    func defaultColor(tokenSet: TableViewCellTokenSet) -> UIColor? {
        switch self {
        case .plain:
            return tokenSet[.cellBackgroundColor].uiColor
        case .grouped:
            return tokenSet[.cellBackgroundGroupedColor].uiColor
        case .clear:
            return .clear
        case .custom:
            return nil
        }
    }
}
```